docs-2.0/7.data-security/2.backup-restore/4.br-restore-data.md

```diff
@@ -16,67 +16,105 @@ To restore data with the BR, do a check of these:
+  ```
+
+  For example, use the following command to upload the backup files in the local `/home/nebula/backup/` to the cluster where the meta service's address is `127.0.0.1:9559`.
 
-   For example, use the following command to restore the data in the local `/home/foesa/backup` path to the cluster where the address is `192.168.*.*:9559` and the user name of the host where the meta/storage service is located is `root`.
+  ```
+  $ ./bin/br restore full --meta "127.0.0.1:9559" --storage "local:///home/nebula/backup/" --name BACKUP_2021_12_08_18_38_08
+  ```
 
-   ```
-   [root]# ./bin/br restore full --meta "192.168.*.*:9559" --storage "local:///home/test/backup" --name "BACKUP_2021_07_29_07_55_08" --user "root"
-   ```
+  Or, you can use the following command to upload the backup files in S3 URL `s3://127.0.0.1:9000/br-test/backup`.
+  ```bash
+  $ ./bin/br restore full --meta "127.0.0.1:9559" --s3.endpoint "http://127.0.0.1:9000" --storage="s3://br-test/backup/" --s3.access_key=minioadmin --s3.secret_key=minioadmin --s3.region="default" --name BACKUP_2021_12_08_18_38_08
+  ```
 
-   If the following information is returned, the data is restored successfully.
-   ```bash
-   restore successed
-   ```
+  If the following information is returned, the data is restored successfully.
+  ```bash
+  Restore succeed.
+  ```
 
-   The parameters are as follows.
+  !!! caution
 
-   | Parameter    | Required | Default value | Description                                                                 | Data type |
-   | ---          | ---      | ---           | ---                                                                         | ---       |
-   | --concurrent | No       | 5             | The maximum number of concurrency for Alibaba Cloud OSS.                    | int       |
-   | --extra_args | No       | None          | The parameters of backup storage tools (OSS/HDFS/S3) for backup.            | string    |
-   | -h,-help     | No       | None          | Checks help for restoration.                                                | -         |
-   | --meta       | Yes      | None          | The IP address and port of the meta service.                                | string    |
-   | --name       | Yes      | None          | The backup name.                                                            | string    |
-   | --storage    | Yes      | None          | The target storage url of BR backup data.                                   | string    |
-   | --user       | Yes      | None          | The username to login into the hosts where meta/storage service is located. | string    |
+        If your new cluster hosts' ip are not all the same with the backup cluster, after restore, you should use `add hosts` to add the Storage hosts in the new cluster one by one.
+
+  The parameters are as follows.
+
+  | Parameter    | Data type | Required | Default value | Description                                                                 |
```

Comment:
please unify the singular form of the first word in the following sentences. e.g. Checks, Specifies, Sets, etc.
and modify desc as the above changes.